package.json

```diff
@@ -40,7 +40,8 @@
     "@react-native/js-polyfills": "^0.75.0",
     "@react-native/normalize-colors": "^0.75.0",
     "@react-native/virtualized-lists": "^0.75.0",
-    "find-babel-config/json5": "^2.1.1"
+    "find-babel-config/json5": "^2.1.1",
+    "react-native-windows": "^0.75.5"
```

Comment:
@tido64 is this change a good way to make sure RNTA uses 0.75.5, instead of publishing this [PR](https://github.com/microsoft/react-native-test-app/pull/2281)?

Reply:
You don't need to add a resolution override. You can bump the package in `yarn.lock`. Since it's already been bumped, you can remove this entry only.

Reply:
Done.